packages/components/src/Chips/InternalChipSingleSelect.tsx

```diff
@@ -0,0 +1,52 @@
+import React from "react";
+import uuid from "uuid";
+import styles from "./InternalChip.css";
+import { InternalChip } from "./InternalChip";
+import { ChipSingleSelectProps } from "./ChipsTypes";
+
+type InternalChipChoiceProps = Pick<
+  ChipSingleSelectProps,
+  "selected" | "onChange" | "children" | "onClickChip" | "name"
+>;
+
+export function InternalChipSingleSelect({
+  children,
+  selected,
+  name = uuid.v1(),
+  onChange,
+  onClickChip,
```

Comment:
Was kinda saving that when, _for some reason_ you want to click the parent and not each chip. Surfaced the callback of clicking the chip on the parent instead of each chip. Either all or nothing type deal.

Reply:
Talked about it. Will change to onClick